Formula/hasura-cli.rb

```diff
@@ -18,7 +18,8 @@ class HasuraCli < Formula
   end
 
   depends_on "go" => :build
-  depends_on "node@16" => :build # Switch back to node with https://github.com/vercel/pkg/issues/1364
+  # At the time of writing, `node` and `node@18` are still unsupported: vercel/pkg#1630
+  depends_on "node@16" => :build
```

Comment:
Should be able to use newer node but `pkg` needs to be updated https://github.com/hasura/graphql-engine/blob/master/cli-ext/package-lock.json#L8211.

Probably should try to get upstream to update. In mean time, could try fetching a newer copy.

Reply:
Thanks for pointing that out; an `npm upgrade pkg` seems to work. I opened https://github.com/hasura/graphql-engine/issues/9440.